server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/MailboxSetMethodContract.scala

```diff
@@ -8080,15 +8080,15 @@ trait MailboxSetMethodContract {
       .subscribeOn(Schedulers.boundedElastic())
       .subscribe()
 
-    Thread.sleep(200)
-    // change subscription
-    JmapRequests.subscribe(mailboxId.serialize())
-
     Awaitility.`with`
       .pollInterval(Duration.ofMillis(100))
       .atMost(Duration.ofSeconds(100))
       .await
       .untilAsserted { () =>
+        // change subscription
+        JmapRequests.subscribe(mailboxId.serialize())
+        Thread.sleep(200)
```

Comment:
This looks weird to me because this change the expected behavior: now we subscribe many times and expect only the first StateChangeEvent.  Anyway...

Reply:
Have a delay time between SSE and eventBus. So, sometimes it pass, sometime it failed.
I think this way will be easier than increase the time sleep.
Or another better way?